File: Supragya/C/reverse_a_number.c
// Day61_Q2.c
// Write a C program to reverse a number.
#include<stdio.h>   
int main(){
    int n;
    printf("Enter the number\n");
    scanf("%d",&n);
    int rev=0;
    while(n>0){
        int rem=n%10;
        rev=rev*10+rem;
        n=n/10;
    }
    printf("The reverse of number is %d",rev);
}

File: Supragya/JAVA/reverse_a_number.java
// Day64_Q1.java
// Write a program to reverse a number using while loop.
import java.util.*;
public class reverse_a_number {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number: ");
        int num = sc.nextInt();
        int rev = 0;
        while(num>0){
            rev = rev*10 + num%10;
            num = num/10;
        }
        System.out.println(rev);
    }
}